api/graphql/schema.graphql

```diff
@@ -142,6 +142,38 @@ type Department {
     name: LocalizedString
 }
 
+enum Status {
+    PENDING
+    DONE
+}
+
+type PoolCandidateFilter {
```

Comment:
I just realize this should also specify a pool. Or array of pools?

Reply:
Array of pools it is.